bin/subiquity-tui

```diff
@@ -66,12 +66,14 @@ def parse_options(argv):
                         help='run in uefi support mode')
     parser.add_argument('--screens', action='append', dest='screens', default=[])
     parser.add_argument('--answers')
+    parser.add_argument('--storage', dest='storage_conf')
     return parser.parse_args(argv)
 
 
 LOGDIR = "/var/log/installer/"
 
 AUTO_ANSWERS_FILE = "/subiquity_config/answers.yaml"
+STORAGE_CONFIG_FILE = "/subiquity_config/subiquity-config-storage.yaml"
```

Comment:
I was originally suggesting that within the answers file you include a 'storage' configuration in the curtin format.
This way users only have a single --answers parameter;  if that input file include a 'storage' config then, subiquity can render that 'storage' section to the subiquity-config-storage.yaml itself.  This prevents users from needing to provide both --answers <my answers> and --storage <my storage config>